### PR TITLE
Replace services service with KIO

### PR DESCRIPTION
**Pull request by eugenkiss**
The https://services.auth.zalando.com will be shut down.

I'm extra careful with the if check.

### CODE REVIEW

**Comment by eugenkiss** [comment]
👍

**Comment by danpersa** [comment]
👍

File: src/main/scala/org/zalando/spearheads/innkeeper/services/team/TeamService.scala [hunk 1]
```diff
@@ -81,10 +81,12 @@ class ZalandoTeamService @Inject() (
   }
 
   private def callApplicationService(applicationName: String, token: String): Future[Result[Team]] = {
-    val url = applicationServiceUrl + applicationName
+    val normalizedApplicationName =
+      if (applicationName.startsWith("stups_")) applicationName.substring("stups_".length) else applicationName
```
Comment (NorthFury) [inline src/main/scala/org/zalando/spearheads/innkeeper/services/team/TeamService.scala]: can be replaced with `val normalizedApplicationName = applicationName.replaceFirst("^stups_", "")`

**Comment by DeTeam** [comment]
👍

File: src/main/scala/org/zalando/spearheads/innkeeper/services/team/TeamService.scala [hunk 2]
```diff
@@ -81,10 +81,12 @@ class ZalandoTeamService @Inject() (
...
+    val url = applicationServiceUrl + normalizedApplicationName
     httpClient.callJson(url, Some(token)).map {
       case jsObject: JsObject =>
-        jsObject.getFields("owner").headOption match {
+        jsObject.getFields("team_id").headOption match {
```
Comment (NorthFury) [inline src/main/scala/org/zalando/spearheads/innkeeper/services/team/TeamService.scala]: I think we should keep backwards compatibility.

**Comment by eugenkiss** [comment]
👍

**Comment by NorthFury** [comment]
:+1:

**Pull request merged**